src/test/java/org/gridsuite/study/server/LoadFlowTest.java

```diff
@@ -0,0 +1,430 @@
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.stream.binder.test.InputDestination;
+import org.springframework.cloud.stream.binder.test.OutputDestination;
+import org.springframework.http.MediaType;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.support.MessageBuilder;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+import static org.gridsuite.study.server.StudyConstants.HEADER_RECEIVER;
+import static org.gridsuite.study.server.notification.NotificationService.HEADER_UPDATE_TYPE;
+import static org.junit.Assert.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * @author Anis Touri <anis.touri at rte-france.com>
+ */
+@RunWith(SpringRunner.class)
+@AutoConfigureMockMvc
+@SpringBootTest
+@DisableElasticsearch
+@ContextConfigurationWithTestChannel
+public class LoadFlowTest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LoadFlowTest.class);
+
+    private static final String CASE_LOADFLOW_UUID_STRING = "11a91c11-2c2d-83bb-b45f-20b83e4ef00c";
+
+    private static final UUID CASE_LOADFLOW_UUID = UUID.fromString(CASE_LOADFLOW_UUID_STRING);
+
+    private static final String NETWORK_UUID_STRING = "38400000-8cf0-11bd-b23e-10b96e4ef00d";
+
+    private static final String LOADFLOW_RESULT_UUID = "1b6cc22c-3f33-11ed-b878-0242ac120002";
+
+    private static final String LOADFLOW_ERROR_RESULT_UUID = "25222222-9994-4e55-8ec7-07ea965d24eb";
+
+    private static final String LOADFLOW_OTHER_NODE_RESULT_UUID = "11131111-8594-4e55-8ef7-07ea965d24eb";
+
+    private static final String LOADFLOW_RESULT_JSON = "{\"resultUuid\":\"0c8de370-3e6c-4d72-b292-d355a97e0d5d\",\"writeTimeStamp\":\"2023-06-13T14:07:30.856705+02:00\",\"componentResults\":[{\"componentResultUuid\":\"4244c84f-855b-4935-a02e-fa34832cf55b\",\"connectedComponentNum\":1,\"synchronousComponentNum\":2,\"status\":\"CONVERGED\",\"iterationCount\":3,\"slackBusId\":\"slackBusId1\",\"slackBusActivePowerMismatch\":4.0,\"distributedActivePower\":5.0},{\"componentResultUuid\":\"83ccb778-1738-40e7-96f8-a341fa054d24\",\"connectedComponentNum\":1,\"synchronousComponentNum\":2,\"status\":\"CONVERGED\",\"iterationCount\":3,\"slackBusId\":\"slackBusId1\",\"slackBusActivePowerMismatch\":4.0,\"distributedActivePower\":5.0}]}\n";
```

Comment:
We should not do that, it's not readable and hard to maintain, can you store it in a JSON file instead ?